src/components/NavbarStudio.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {FC, useState} from 'react';
+import './NavbarStudio.scss';
+import NavbarSVGSelector from './NavbarStudioSVGSelector';
+
+const NavbarStudio: FC = () => {
+    const [isMenuActive, setIsMenuActive] = useState(false);
+    const [isButtonPressed, setIsButtonPressed] = useState(0);
+    const items = [
+    {id: 1, value: 'Menu', href: '#!', icon: 'menu'},
+    {id: 2, value: 'Close', href: '#!', icon: 'close'},
+    {id: 3, value: 'My lesson', href: '#!', icon: 'my_lesson'},
+    {id: 4, value: 'Management', href: '#!', icon: 'management'},
+  ]
+
+  let imageWrapper = !isMenuActive ? 'image__wrapper--active' : '';
+  let svg = isMenuActive ? 'svg--active' : '';
```

Comment:
Please, rename this constant to svgClassName, as the svg often means svg itself. This name will be clearly to understand.

Also, note, that you can use classnames as they are already attached to the project: https://www.npmjs.com/package/classnames

Reply:
done